commodore/compile.py

```diff
@@ -60,24 +62,16 @@ def _regular_setup(config, cluster_id):
     customer_id = cluster['tenant']
 
     # Fetch components and config
-    try:
-        _fetch_global_config(config, cluster)
-        _fetch_customer_config(config, customer_id)
-        fetch_components(config)
-    except Exception as e:
-        raise click.ClickException(f"While cloning git repositories: {e}") from e
-
+    _fetch_global_config(config, cluster)
```

Comment:
Here you are removing the gift wrapping of exceptions. For the function `_relsymlink`. What is the thinking?

Reply:
The wrapping was to generic and added wrong/confusing error messages to unrelated exceptions